lib/logger/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as Logger from 'bunyan';
+
+const is = require('@sindresorhus/is');
+const PrettyStdout = require('./pretty-stdout').RenovateStream;
+const configSerializer = require('./config-serializer');
+const errSerializer = require('./err-serializer');
+const cmdSerializer = require('./cmd-serializer');
+
+let meta = {};
+
+const stdout: Logger.Stream = {
+  name: 'stdout',
+  level: (process.env.LOG_LEVEL as Logger.LogLevel) || 'info',
+  stream: process.stdout,
+};
+
+if (process.env.LOG_FORMAT !== 'json') {
+  const prettyStdOut = new PrettyStdout();
+  prettyStdOut.pipe(process.stdout);
+  stdout.stream = prettyStdOut;
+}
+
+const bunyanLogger: any = Logger.createLogger({
+  name: 'renovate',
+  serializers: {
+    body: configSerializer,
+    cmd: cmdSerializer,
+    config: configSerializer,
+    err: errSerializer,
+  },
+  streams: [stdout],
+});
+
+const logFactory = (level: string): any => {
```

Comment:
we should define a proper interface as return type, see here for bunyan definition:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3fa33b4f74361d7a77e43f1d45b216dc4b8e3786/types/bunyan/index.d.ts#L34-L54

Reply:
True, but also see the discussion around this here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/21080

Reply:
Interesting. I prefer to always be strict because we can find a lot of bugs at compile time then. Here I would make some temporary simplifications while converting because it will need all lot of time to cleanup such a huge codebase.

Reply:
Yeah, I agree that strictness is definitely useful. I'll come back to this at some point, when we're further along with the conversion.